README.md

```diff
@@ -2,5 +2,73 @@
 
 # EGI trust anchors container
 
-This is a [CENTOS 7 utility container](https://hub.docker.com/r/indigoiam/egi-trustanchors) that contains fetch-crl and other utilities
-to provide up-to-date trust anchors to relying applications.
+This is [container](https://hub.docker.com/r/indigoiam/egi-trustanchors) contains fetch-crl and other utilities
+to provide up-to-date trust anchors to relying applications, like Nginx, and to VOMS Attribute Authority service. When run,
+the container updates 2 components:
+
+- The system trust anchors, known as CA bundle, typically found in /etc/pki on an EL-based system
+- The EGI trust anchors maintained by fetch-crl and typically found in /etc/grid-security/certificates
+
+These 2 components are exported as 2 separate volumes mapped when the container is launched and used as volumes read by other
+containers needing these informations.
+
+The container is currently based on AlmaLinux 9.
+
+
+## Configuration
+
+The main configuration needed is the declaration of the 2 volumes. To avoid that the volumes are destroyed when the container is
+deleted, and thus become unavailable to the containers using them, it is recommended to create the volumes before launching
+the containers and to declare the volumes as `external` in the Docker/Podman configuration. It is typically done with (volume names are
+free but must be consistent across the configuration):
+
+```bash
+# podman command can be used if using Podman instead of Docker
+docker volume create cabundle
+docker volume create trustanchors
+```
+
+It is also necessary to define the following environment variable when launching the container so that fetch-crl is run as startup:
+
+```
+FORCE_TRUST_ANCHORS_UPDATE=1
+```
+
+
+## Compose file
+
+Wheter using Docker or Podman, it is recommended to use a compose file to declare the container and how to run it. A typical
+compose section for running this container would be:
+
+```yaml
+services:
+    trustanchors:
+        image: docker.io/indigoiam/egi-trustanchors:main
+        container_name: egi-trustanchors
+        environment:
+            - FORCE_TRUST_ANCHORS_UPDATE=1
+        volumes:
+                - trustanchors:/etc/grid-security/certificates
+                - cabundle:/etc/pki
+
+volumes:
+    cabundle:
+        external: true
```

Comment:
In our compose files as far as I know we have never used this `external` directive. We tend to use compose files to run test suites in Continuous Integration, where the volumes are created on the fly and then removed. Would you add a note for this typical use, saying that `external` is not needed?

Reply:
Maybe I was not clear when I mentioned the use of a compose during a CI job. As you can see for example in the [VOMS testsuite](https://github.com/italiangrid/voms-testsuite/blob/master/compose/docker-compose.ci.yml), the compose includes many services, some of them using the EGI trustanchors volume, which is initialized by this container. In that situation the volume lives only for the duration of the compose run and that's fine.

Reply:
So I think we agree! It is basically what I wrote: in a CI it may not make sense to use `external: true` but in a production service relying on Podman/Docker you really don't want your volume to be destroyed: you just want it to be updated. This this parameter is required.